README.md

```diff
@@ -1,17 +1,25 @@
-# Hello World
+![Python Template - Hello World Banner](https://raw.githubusercontent.com/linz/template-python-hello-world/badges/docs/_static/banner.png)
 
-[![Build Status](https://github.com/linz/template-python-hello-world/workflows/Build/badge.svg)](https://github.com/linz/template-python-hello-world/actions)
-[![Code Style](https://img.shields.io/badge/Code%20Style-black-000000.svg)](https://github.com/psf/black)
-[![License](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/linz/template-python-hello-world/blob/master/LICENSE)
+<h2 align="center">A minimal template for Python development</h2>
 
-A minimal template for Python development.
+<p align="center">
+<a href="https://github.com/linz/template-python-hello-world/actions"><img alt="GitHub Actions Status" src="https://github.com/linz/template-python-hello-world/workflows/Build/badge.svg"></a>
```

Comment:
I am not entirely sold on putting HTML into your markdown files, 

Is there a reason you went away from the `[![X](img)](link)` syntax

Reply:
Centred header / badges look nice.

Not worth it for a minor visual difference though. Reverted back to Markdown.